### DISCUSSION TITLE
How to find out if database is MySQL or PostgreSQL in middleware?

### DISCUSSION THREAD

**Discussion by janpio** [prisma/prisma#6408] [Q&A]
I want to write a middleware that triggers a raw query. Of course that query should be different per database used, or even throw an error if run for an unsupported database. How can I find out inside my middleware if it is being run with a MySQL or PostgreSQL database?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
